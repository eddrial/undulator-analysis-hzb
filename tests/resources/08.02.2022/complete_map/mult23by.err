 error report for the data set mult23by.dvm        
--------------------------------------------
      
         127   data points thrown away to get a
         monotonically increasing data set
 the truncated data set has only   1591 data points 
        out of totally   2558 data points
 the rms value of the residuals is  0.54337E-02
 the maximum residual is  0.18795E-01
 the rms value of the lambda/2-filtered data is  0.35673E-01
 the maximum value of the lambda/2-filtered data is  0.27221E+00
