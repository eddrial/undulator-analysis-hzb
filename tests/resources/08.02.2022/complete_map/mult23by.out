mult23by.dvm
            

 names of parameters as follows : 
b(1) == amplitude, 1st harm.
b(2) == amplitude, 3rd harm.
b(3) == amplitude, 5rd harm.
....                        
b(n) == amplitude, n^th harm.
b(n+1) == lambda, mm.
b(n+2) == x0, 1st harm, mm.
b(n+3) == x0, 3rd harm, mm.
b(n+4) == x0, 5th harm, mm.
....
b(nparm) == x0, n^th harm, mm.

 parameters:
   1  1.0068206E+00
   2 -1.0319143E-02
   3 -9.7230831E-03
   4 -1.4212285E-04
   5 -2.5530264E-04
   6  5.5981308E+01
   7  1.0438510E+03
   8  1.0435944E+03
   9  1.0439200E+03
  10  1.9210041E-01
  11 -1.6634651E+01
 number of iterations:    9
     sumsq:  4.69740E-02
       rms:  5.43368E-03  dimension of magnetic field
       rms:  5.39683E-01  percent of b_eff
   rms_ext:  5.18668E-01  percent of b_eff
 B_eff =   1.0068E+00
 B_max =   9.8722E-01
 K_eff =    5.266
 lamda_1 (nm) =     37.76379

        eigenvectors in parameter space

 error                         direction
     1.73,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00
     0.75,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00
     0.06,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00
     0.04,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00
     0.00,  0.00  0.00  0.00  0.00  0.00 -0.05  1.00  0.00  0.00  0.00  0.00
     0.00,  0.65 -0.44 -0.47 -0.26 -0.29 -0.08  0.00  0.00  0.00  0.00  0.00
     0.00,  0.42  0.46  0.57 -0.49 -0.24  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.57  0.45 -0.14  0.60  0.30  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.20 -0.36  0.25 -0.28  0.83 -0.03  0.00  0.00  0.00  0.00  0.00
     0.00,  0.16 -0.52  0.61  0.51 -0.28 -0.04  0.00  0.00  0.00  0.00  0.00
     0.00,  0.06 -0.07  0.00 -0.01 -0.01  0.99  0.05  0.00  0.00  0.00  0.00
   
  calculated kicks for trajectory compensation: 
 kick_1:  -0.01230 Tmm
 kick_2:   0.01230 Tmm
      
 the rms value of the second derivative is  0.00000E+00
 the maximum value of the second derivative is  0.00000E+00
      
 the maximum transverse movement of the electron in mu_meter
                  (peak to peak) is  0.33926E+02
      
 the truncated data set has only   1591 data points 
      
 the rms value of the residuals is  0.54337E-02
 the maximum residual is  0.18795E-01
      
 the rms value of the lambda/2-filtered data is  0.35673E-01
 the maximum value of the lambda/2-filtered data is  0.27221E+00
      
 the rms phase error (all data) is    2.858 deg.
 the maximum phase error (all data) is    9.540 deg.
      
 the rms phase error (extrema) is    3.432 deg.
 the maximum phase error (extrema) is    9.542 deg.
      
 the Walker-rms phase error (all data) is   19.174 deg.
 the Walker-maximum phase error (all data) is   34.781 deg.
      
 the Walker-rms phase error (extrema) is    3.632 deg.
 the Walker-maximum phase error (extrema) is    9.544 deg.
  
  phase: straight line subtracted
 -----------------------------------------
 total phase error             2.857760    
 local phase error             2.851930    
 slowly varying phase error  NaN           
 oscillatory phase error      0.8842436    
 -----------------------------------------
